express/code/blocks/frictionless-quick-action/frictionless-quick-action.css

```diff
@@ -52,13 +57,18 @@
     flex-wrap: wrap;
 }
 
+.frictionless-quick-action .fqa-container .button-container.hero-animation-overlay {
+    width: var(--frictionless-quick-action-video-width);
+    height: var(--frictionless-quick-action-video-height);
```

Comment:
similarly maybe use min-height

Reply:
Thank you @JingleH . Can you review / approve?